### DISCUSSION TITLE
"SumatraPDF-settings.txt" cannot be changed.

### DISCUSSION THREAD

**Discussion by yahaha6** [sumatrapdfreader/sumatrapdf#2633] [Q&A]
I want to change the background color of the UI，and changed the value of  "FixedPageUI.BackgroundColor" in "SumatraPDF-settings.txt" to #ff0000.The interface of SumatraPDF did not change. After the software is restarted, the "SumatraPDF-settings.txt" file changes to the default.

<img width="218" alt="Snipaste_2022-05-30_18-01-55" src="https://user-images.githubusercontent.com/46367705/170969491-a58013a1-131f-46ad-8cf0-1b8d75fc1f85.png">

Why can't I successfully modify "SumatraPDF-settings.txt"?
Thank you!

**Comment by kjk** [comment]
You can change `SumatraPDF-settings.txt`.

2 things:
* `BackgroundColor` is for the background color of rendered documents `MainWindowBackground` is for the color of the window
* if sumatra restores the setting to default value, most likely you made a typo and entered incorrect value
